Academic authorship chapter for Guide for Communication (#2063)
alan-turing-institute/the-turing-way: book/website/afterword/glossary.md

FAIR
  Findable, Accessible, Interoperable and Reusable.
  
First author

>  The most prominent position in academic authorship. It conveys this persons position as the researcher than has made the greatest contribution to the research. 

Author: ```suggestion
 The most prominent position in academic authorship. It conveys this person's position as the researcher who has made the greatest contribution to the research. 
```

GitLab
  GitLab is a web-based DevOps lifecycle tool that provides a Git-repository manager providing wiki, issue-tracking and continuous integration and deployment pipeline features, using an open-source license, developed by GitLab Inc.
  
Ghost author
 It is a person who writes an academic article without having carried out the research. It could be a professional writer. They would often not qualify as an author under the ICMJE criteria for authorship.
 
Gift author

>  People who are listed as authors but who did not make significant contribution to the research. This is also known as a guest author.

Author: ```suggestion
 People who are listed as authors but who did not make significant contributions to the research. This is also known as a guest author.
```

 
Group authorship
 Some journals permit the use of group names but many require contributors to be listed and/or the writing group to be named. This is the same as shared authorship.
 
Guarantor

>  As well as fullfilling criteria for being a named author, some journals require one or more authors that take responsibility for the integrity of the work as a whole from inception to published article.

Author: ```suggestion
 As well as fulfilling criteria for being a named author, some journals require one or more authors that take responsibility for the integrity of the work as a whole from inception to the published article.
```

  Documenting research code and routines, and making them freely accessible and available.

ORCID

>   Open Researchers and Contibutor iD. It is a long lasting unique identifier for you as a researcher. A peristent digital identifier for researchers that can be used on publications to ensure fair credit is given for all the researchers works. 

Author: ```suggestion
  Open Researchers and Contributor iD. It is a long lasting unique identifier for you as a researcher. A persistent digital identifier for researchers' that can be used on publications to ensure fair credit is given for all the researchers works. 
```

SHA
  Unique string of numbers of letters used to identify every commit or node in the repository.

Shared authorship
 Some journals permit the use of group names but many require contributors to be listed and/or the writing group to be named. This is the same as group authorship.

Smoke Testing

>   Very brief initial checks that ensures the basic requirements required to run the project hold. If these fail there is no point proceeding to additional levels of testing until they are fixed.

Author: ```suggestion
  Very brief initial checks that ensure the basic requirements required to run the project hold. If these fail there is no point proceeding to additional levels of testing until they are fixed.
```
